centreon/www/install/createTables.sql

```diff
@@ -2638,6 +2638,63 @@ CREATE TABLE IF NOT EXISTS `dashboard_widgets` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+CREATE TABLE IF NOT EXISTS `dashboard_playlist` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `description` TEXT NULL,
+  `rotation_time` TINYINT UNSIGNED NOT NULL,
+  `created_at` INT(11) UNSIGNED NOT NULL,
+  `updated_at` INT(11) UNSIGNED NULL,
+  `created_by` INT NULL,
+  `updated_by` INT NULL,
+  `is_public` TINYINT(1) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY (`name`),
+  CONSTRAINT `dashboard_playlist_author_id`
+    FOREIGN KEY (`created_by`)
+    REFERENCES `contact` (`contact_id`) ON DELETE SET NULL,
+  CONSTRAINT `dashboard_playlist_editor_id`
+    FOREIGN KEY (`updated_by`)
+    REFERENCES `contact` (`contact_id`) ON DELETE SET NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE IF NOT EXISTS `dashboard_playlist_relation` (
+  `dashboard_id` INT UNSIGNED NOT NULL,
+  `playlist_id` INT UNSIGNED NOT NULL,
+  `order` INT(11) NOT NULL,
+  UNIQUE KEY(`dashboard_id`, `playlist_id`),
+  CONSTRAINT `dashboard_playlist_relation_dashboard_id`
+    FOREIGN KEY (`dashboard_id`)
+    REFERENCES `dashboard` (`id`) ON DELETE CASCADE,
+  CONSTRAINT `dashboard_playlist_relation_playlist_id`
+    FOREIGN KEY (`playlist_id`)
+    REFERENCES `dashboard_playlist` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```

Comment:
OPTIONAL
Add a constraint to make order value unique for a playlist_id

```suggestion
CREATE TABLE IF NOT EXISTS `dashboard_playlist_relation` (
  `dashboard_id` INT UNSIGNED NOT NULL,
  `playlist_id` INT UNSIGNED NOT NULL,
  `order` INT(11) NOT NULL,
  UNIQUE KEY(`dashboard_id`, `playlist_id`),
  CONSTRAINT `AK_PlaylisId_Order` UNIQUE (`playlist_id`, `order`),
  CONSTRAINT `dashboard_playlist_relation_dashboard_id`
    FOREIGN KEY (`dashboard_id`)
    REFERENCES `dashboard` (`id`) ON DELETE CASCADE,
  CONSTRAINT `dashboard_playlist_relation_playlist_id`
    FOREIGN KEY (`playlist_id`)
    REFERENCES `dashboard_playlist` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```

Reply:
the order is for a dashboard in a playlist, maybe extends the unique key to handle the 3 columns should be better to ensure unicity